package.json

```diff
@@ -55,7 +55,8 @@
     "semantic-release": "^6.3.6"
   },
   "dependencies": {
-    "brcast": "^2.0.0"
+    "brcast": "^2.0.0",
+    "prop-types": "^15.5.10"
```

Comment:
Normally we might opt to put this in `peerDependencies`, but since it's an *optional* dependency, let's follow [glamorous](https://github.com/paypal/glamorous/blob/master/package.json#L68) and place this in `devDependencies`